main_test.go

```diff
@@ -60,12 +60,17 @@ func TestRunWithOptions(t *testing.T) {
 		Procedure:  fooMethod,
 	}
 
+	blockedCallerNames = map[string]struct{}{
```

Comment:
should we be resetting this back to the original value?
options:
- Don't set this value at all, and assume "tcurl" is in that list
- Set this value early in the code (maybe in `init` in the `main_test.go` so we know all tests use this value)
- Reset back to original.

I'm learning to one of the earlier options.